features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/site/public/pages/README.md

```diff
@@ -0,0 +1,22 @@
+<!--
+ * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Let's change the year to 2020.

Reply:
fixed in https://github.com/wso2/carbon-apimgt/pull/9045/commits/b857d16f62df6be3152b273bd00333653436d8b0